src/cryptography/hazmat/bindings/openssl/_conditional.py

```diff
@@ -125,7 +125,6 @@ def cryptography_has_tlsv13_functions() -> typing.List[str]:
         "SSL_set_post_handshake_auth",
         "SSL_SESSION_get_max_early_data",
         "SSL_write_early_data",
-        "SSL_read_early_data",
```

Comment:
Why was this removed?

Reply:
Mistake! Will fix.

Reply:
[021f7ed](https://github.com/pyca/cryptography/pull/8946/commits/021f7ed13342e6a87f5edd0ceccf473f58503af8)